### ISSUE TITLE
As a red-green color blind user, I cannot see the indicator for mandatory fields

### ISSUE THREAD

**Issue by mizpeggy** [carbon-design-system/carbon#3433]
## Environment

> Operating system

> Browser

> Automated testing tool and ruleset

> Assistive technology used to verify

## Detailed description

> What version of the Carbon Design System are you using?

> What did you expect to happen?

> What happened instead?

> What WCAG 2.1 checkpoint does the issue violate?

## Steps to reproduce the issue

1. Step one
2. Step two
3. Step three
4. etc.

> Please create a reduced test case in CodeSandbox
>
> - Style and vanilla JS:
>   https://codesandbox.io/s/github/carbon-design-system/carbon/tree/master/packages/components/examples/codesandbox
> - React:
>   https://codesandbox.io/s/github/carbon-design-system/carbon/tree/master/packages/react/examples/codesandbox

## Additional information

- Screenshots or code
- Notes

**Comment by laurenmrice** [comment]
not stale

**Comment by mizpeggy** [comment]
This is an Accessibility issue.  I'm not a developer, and I've heard that
maybe some of your later releases have addressed this issue.  However, if
any of your users are using this release of Carbon and encounter issues,
any legal action, or anything else, and they come to you . . . it looks
like you closed an issue without any references, redirects to later
versions, or remediation.  It makes your organization look kind of
non-caring.  I wish I were a developer as I could have been more active.
Please do the right thing here.  Thank you.

**Comment by tw15egan** [comment]
Was incorrectly closed as a stale issue, reopened so we can resolve this. 

cc @carbon-design-system/design

**Comment by laurenmrice** [comment]
@mizpeggy can you clarify which field, and could you provide some additional screenshots or images of what you are referring to? Are you talking about an asterisks symbol?

**Comment by chrisconnors-ibm** [comment]
@mizpeggy none of the template items that allow us to diagnose and troubleshoot your issue have been filled out. Without the **Environment**, **Detailed description**, **Steps to reproduce** (where did you see the problem and how did you get there) _we cannot reproduce it_. It's not a question of "doing the right thing", we simply don't know what your issue actually _is_. Without that information, we'll be forced to close this issue as it is: _Cannot Reproduce_

Our [Forms guidance is _all fields are required_](https://www.carbondesignsystem.com/components/form/usage#validation--errors) and then uses 3 cues _other than color_ to signify error:
1) Stroke around input field
2) label below "This is Required"
3) Error Icon in the field

<img width="317" alt="Screen Shot 2019-08-23 at 2 57 30 PM" src="https://user-images.githubusercontent.com/35537391/63617747-3ed03e80-c5b8-11e9-9dc2-7b8a4236fce5.png">

Or without the red:
<img width="317" alt="no color" src="https://user-images.githubusercontent.com/35537391/63618004-ef3e4280-c5b8-11e9-9937-4d6b3e1d73a8.png">


There _would_ be an accessibility issue if we _only_ used color to signify error.

Our guidance is to mark Optional Fields, and no color is involved:
![image](https://user-images.githubusercontent.com/35537391/63617614-e731d300-c5b7-11e9-8daf-f379d23c1693.png)
![image](https://user-images.githubusercontent.com/35537391/63617779-57d8ef80-c5b8-11e9-8a2e-0e4f29c7f187.png)

**Comment by chrisconnors-ibm** [comment]
I'm going to leave this open for a week to give @mizpeggy a chance to submit the details we need to reproduce.

**Comment by mizpeggy** [comment]
Thank you.  I am glad you are using the optional marking.  However this
really throws the mid/late career men I work with (this is the demographic
that is overly red/green color blind) because they just do not pay attn to
form field designations.  And we certainly aren't ever going to do anything
about that!  On the form we are using for a project at work, we have also
placed the words (mandatory field) plus the red color designation.  That
seems to help.
thanks for you kind attention.
